libs/sdk-ui-kit/styles/scss/variables.scss

```diff
@@ -43,9 +43,19 @@ $gd-palette-primary-base-darken06: var(--gd-palette-primary-base-d06, darken($gd
 // secondary colors
 $gd-color-positive-dimmed: mix($gd-color-positive, $gd-color-text-light, 10%);
 $gd-color-negative-dimmed: mix($gd-color-negative, $gd-color-text-light, 10%);
-$gd-color-warning-dimmed: mix($gd-color-warning, $gd-color-text-light, 20%);
-$gd-color-warning-label-dimmed: mix($gd-color-warning, $gd-color-text, 40%);
-$gd-color-warning-label: mix($gd-color-warning, $gd-color-text, 20%);
+
+// warning colors
+$gd-palette-warning-base: var(--gd-palette-warning-base, $gd-color-warning);
+$gd-palette-warning-dimmed: var(
+    --gd-palette-warning-dimmed,
+    mix($gd-color-warning, $gd-color-text-light, 20%)
+);
+$gd-palette-warning-base-t50: var(--gd-palette-warning-base-t50, transparentize($gd-color-warning, 0.5));
+$gd-color-warning-label: var(--gd-palette-warning-dimmed, mix($gd-color-warning, $gd-color-text, 20%));
```

Comment:
this is using the same --gd value, but different default:
mix($gd-color-warning, $gd-color-text-light, 20%) VS
mix($gd-color-warning, $gd-color-text, 20%)
so it should have probably different --gd variable filled differently in TS code